migrations/etl/1_Mine_profile_detail_location.sql

```diff
@@ -147,6 +150,43 @@ BEGIN
         (new.lat_dec <> 0 AND new.lon_dec <> 0);
     SELECT count(*) FROM mine into new_row;
     SELECT count(*) FROM mine_location into location_row;
+
+    -- Upsert data from new_record into mine_type
+    WITH new_record AS (
+        SELECT *
```

Comment:
It looks like sometimes we * here and sometimes we list the columns explicitly. I prefer the latter especially when we start mapping them below.

Reply:
I can select the columns explicitly if that's the pattern we want to follow